fragments/master.sh

```diff
@@ -105,8 +113,46 @@ detect_lsb() {
 DOCKER_CONF=""
 
 start_k8s(){
+
+	# Configure docker settings, then restart it
+    case "${lsb_dist}" in
```

Comment:
This code is similar to set-registry function. Also, we would reboot docker twice. Code with similar functionality should be closer co-located. I would merge or refactor the two functions or refactor somehow else